charts/castai-kvisor/values.yaml

```diff
@@ -41,6 +46,12 @@ castai:
   # CASTAI public api url.
   apiURL: "https://api.cast.ai"
 
+  # kvisor admission service configuration.
+  serviceName: "kvisor"
```

Comment:
I think these variables are better suited for policyEnforcement section. In castai section we usually just put castai api related config.